UnityMcpBridge/UnityMcpServer~/src/tools/manage_editor.py

```
@@ -1,37 +1,28 @@
-        "- layer_name: Layer name for specific actions\n\n"
-        "Returns:\n"
-        "Dictionary with operation results ('success', 'message', 'data')."
-    ))
+    @mcp.tool(name="manage_editor", description="Controls and queries the Unity editor's state and settings")
     @telemetry_tool("manage_editor")
     def manage_editor(
         ctx: Context,
-        action: str,
-        wait_for_completion: bool = None,
+        action: Annotated[Literal["telemetry_status", "telemetry_ping", "play", "pause", "stop", "get_state", "get_project_root", "get_windows",
+                                  "get_active_tool", "get_selection", "get_prefab_stage", "set_active_tool", "add_tag", "remove_tag", "get_tags", "add_layer", "remove_layer", "get_layers"], "Operations"] = None,
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)